install/install_for_windows.ps1

```diff
@@ -1,90 +1,147 @@
-$version_to_use=$env:TIPI_INSTALL_VERSION
+
+
+$version_to_use = $env:TIPI_INSTALL_VERSION
+$INSTALL_FOLDER = $env:TIPI_INSTALL_DEST
+
+$system_install = $false
+if([bool]::TryParse($env:TIPI_INSTALL_SYSTEM, [ref]$system_install)) {
+    Write-Output "INFO: TIPI_INSTALL_DEST set, intepreted as boolean '$system_install'"
+}
 
 if ([string]::IsNullOrEmpty($version_to_use)) {
-$version_to_use="v0.0.31"
+    $version_to_use="v0.0.31"
+}
+
+if ([string]::IsNullOrEmpty($INSTALL_FOLDER)) {
+    $INSTALL_FOLDER = Join-Path -Path ([Environment]::GetFolderPath('LocalApplicationData')) -ChildPath "\tipi"
+}
+
+if($system_install) {
+    $INSTALL_FOLDER = "C:\ProgramData\tipi"
 }
-$INSTALL_FOLDER="C:\ProgramData\tipi"
-$TIPI_EXE="$INSTALL_FOLDER\tipi.exe"
-$TIPI_URL="https://github.com/tipi-build/cli/releases/download/$version_to_use/tipi-$version_to_use-windows-win64.zip"
+

```

Comment:
If I set TIPI_INSTALL_DEST + TIPI_INSTALL_SYSTEM I end up installed in ProgramData nonetheless.